Content.Client/UserInterface/Systems/MenuBar/Widgets/GameTopMenuBar.xaml

```diff
@@ -11,25 +11,29 @@
            Orientation="Horizontal"
            HorizontalAlignment="Stretch"
            VerticalAlignment="Top"
-           SeparationOverride="5"
 >
+    <GridContainer
+        Rows="2"
+        HSeparationOverride="5"
+        VSeparationOverride="5"
+        HorizontalExpand="True">
```

Comment:
Is the bulging buttons taking up the chat window intentional? 
![image](https://github.com/user-attachments/assets/f2825f7e-89e0-4c08-863c-85c0199891f6)

Reply:
thats how it used to be just less wide, can be reverted